exercises/storeWithArrays.js

```diff
@@ -48,7 +30,11 @@ function newSofa(name, price) {
  * and `false` otherwise.
  */
 function userCanAffordSofa(user, sofa) {
-  // This is your job. :)
+  if (sofa[1] > user[2]) {
```

Comment:
This really threw me off. At first I thought you had the logic backwards, but then I see you're returning `false` is this expression is `true`. I wrote an initial set of comments only to think that maybe you're returning `false` in the first branch.

You can just write:

```javascript
function userCanAffordSofa(user, sofa) {
  return sofa[1] <= user[2];
}
```